### PR TITLE
Initial comments for Code Review Party 2020

### PR DESCRIPTION
**Pull request by calculusrobotics**
Craig I am so sorry

### CODE REVIEW

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/BitBucketSubsystem.java [hunk 1]
```diff
@@ -114,7 +114,10 @@ public void initialize() {
 
     public abstract void disable();
 
+    // list of Talons this subsystem uses, used in CANChecker
     protected List<BaseTalon> talons = new ArrayList<BaseTalon>();
+    // add talons to talons list
     public abstract void listTalons();
```
[the remark this answers is not in the document]
Comment (calculusrobotics) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/BitBucketSubsystem.java]: copy-paste time

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java [hunk 2]
```diff
@@ -41,74 +41,105 @@
...
     private SpeedControllerGroup leftGroup;
     private SpeedControllerGroup rightGroup;
 
+    // differential drive representing our drive base
     private DifferentialDrive differentialDrive;
 
+    /**
+     * Create an instance of the DriveSubsystem
+     * 
+     * @param config configuration file to use
```
[the remark this answers is not in the document]
Comment (calculusrobotics) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java]: but its in a fileeee

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java [hunk 3]
```diff
@@ -541,7 +600,7 @@ public void periodic(float deltaTime) {
                 driveMethod = DriveMethod.VELOCITY;
             }
 
-            // won't happen by AZ North
+            // won't happen by AZ North (and now AZ North won't happen!)
```
[the remark this answers is not in the document]
Comment (calculusrobotics) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java]: we live in a society

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java [hunk 4]
```diff
@@ -742,6 +785,7 @@ public SimpleMotorFeedforward getCharacterization() {
     public PIDController getLeftAutoPID() { return leftAutoPID; }
     public PIDController getRightAutoPID() { return rightAutoPID; }
 
+    // command a voltage to both motors, currently only being used to stop both after a trajectory is finished
```
[the remark this answers is not in the document]
Comment (calculusrobotics) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java]: oh oops you're right. my brain was still thinking as if we used the TalonFX velocity mode but then we switched to PID!

File: InfiniteRecharge/src/main/java/frc/robot/utils/data/filters/statespace/GenericLuenbergerObserver.java [hunk 5]
```diff
@@ -8,6 +8,13 @@
 
 
 
+/*
+ * ==================================
+ * 2020 Code Review Party Information
+ * 
+ * Feel free to ignore this file
```
Comment (cpostbitbuckets) [inline InfiniteRecharge/src/main/java/frc/robot/utils/data/filters/statespace/GenericLuenbergerObserver.java]: Please change this to: don’t ignore this one. Look upon my mighty generics and cower!

Comment (cpostbitbuckets) [reply InfiniteRecharge/src/main/java/frc/robot/utils/data/filters/statespace/GenericLuenbergerObserver.java]: Seriously though, well done.

**Pull request merged**